splitelf.c

```diff
@@ -0,0 +1,56 @@
+#include <sys/types.h>
```

Comment:
This I can do though it is short.

Any restriction of license with flathub? Can it be accepted if I choose GPL3?